04.ls/wc.rb

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  cmd_arg = receive_command_argument
+  opt_params = cmd_arg[0]
+  argv = cmd_arg[1]

```

Comment:
```suggestion
  opt_params, argv = receive_command_argument
```

Rubyだとこんなふうに引数を受け取れます。

ただ、opt_paramsやargvという変数名だと中身がわかりづらいので、options, files（もしくはpaths）みたいな名前にした方がわかりやすいと思います

Reply:
命名の変更と受け取り方を変更しました。